### DISCUSSION TITLE
求帮助 谁能解答下 红线的数值都代表什么意思

### DISCUSSION THREAD

**Discussion by m8761731** [NebuTech/NBMiner#580] [Q&A]
![阿里旺旺图片20211102092007](https://user-images.githubusercontent.com/93560594/139772417-fbf870c1-0bbd-45cc-9889-ec74159bbaff.j
部分80卡还在用38.2的内核。

新人求解答 新的版本这些数值代表什么。

目前超频工具用的轻松矿工

**Comment by m8761731** [comment]
图片没传上去
Reject  Inv  Powr CTmp  MTmp  CC1k  GMC1K  MUt1  
这些都代表什么？

**Comment by yueliang0329** [comment]
拒绝 、无效、功耗、核心温度、显存温度、核心锁、显存锁 ，最后一个我也不知道
